api_v3/services/ResponseProfileService.php

```diff
@@ -188,4 +188,30 @@ function recalculateAction(KalturaResponseProfileCacheRecalculateOptions $option
 	{
 		return KalturaResponseProfileCacher::recalculateCacheBySessionType($options);
 	}
+	
+	/**
+	 * List response profiles by filter and pager
+	 * 
+	 * @action clone
+	 * @param int $id
+	 * @throws KalturaErrors::RESPONSE_PROFILE_ID_NOT_FOUND
+	 * @return KalturaResponseProfile
+	 */
+	function cloneAction ($id)
+	{
+		$origResponseProfileDbObject = ResponseProfilePeer::retrieveByPK($id);
+		if (!$origResponseProfileDbObject)
+			throw new KalturaAPIException(KalturaErrors::RESPONSE_PROFILE_ID_NOT_FOUND, $id);
+			
+		$newResponseProfileDbObject = $origResponseProfileDbObject->copy();
+		$newResponseProfileDbObject->setSystemName($origResponseProfileDbObject->getSystemName() . '_' . uniqid());
+		
+		$newResponseProfile = new KalturaResponseProfile();
+		$newResponseProfileDbObject = $newResponseProfile->toInsertableObject($origResponseProfileDbObject);
```

Comment:
toInsertableObject doesn't make sense here, that method should validate for insert and then call toObject.
The validate doesn't make sense because it's validating the KalturaResponseProfile which is empty in this case and the toObject populate the $origResponseProfileDbObject which already filled in this case.